lib/pages/home/home_page.dart

```diff
@@ -7,23 +7,48 @@ import 'package:survey/pages/home/home_controller.dart';
 import 'package:survey/pages/home/survey_carousel/survey_carousel_card.dart';
 import 'package:survey/pages/home/survey_carousel/survey_ui_model.dart';
 
+// TODO: use the real data in the Integrate ticket
+var cards = [
+  SurveyUiModel(
+    id: "111",
+    title: "Mock Title: Scarlett Bangkok ",
+    description:
+        "Mock Description: We'd love ot hear from you! Or a very long paragraph here to test the UI display",
+    imageUrl: "https://dhdbhh0jsld0o.cloudfront.net/m/1ea51560991bcb7d00d0_l",
+  ),
+  SurveyUiModel(
+    id: "222",
+    title: "Mock Title 2: ibis Bangkok Riverside",
+    description: "Mock Description 2: We'd love to hear from you!",
+    imageUrl: "https://dhdbhh0jsld0o.cloudfront.net/m/287db81c5e4242412cc0_l",
+  ),
+  SurveyUiModel(
+    id: "333",
+    title: "Mock Title 3: ibis Bangkok Riverside",
+    description: "Mock Description 3: We'd love to hear from you!",
+    imageUrl: "https://dhdbhh0jsld0o.cloudfront.net/m/0221e768b99dc3576210_l",
+  ),
+];
+
 class HomePage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       body: GetBuilder(
         init: HomeController(),
         builder: (controller) => Stack(children: [
-          SurveyCarouselCard(
-            // TODO: implement the carousel list next and remove these
-            surveyUiModel: SurveyUiModel(
-              id: "111",
-              title: "Mock Title: Scarlett Bangkok ",
-              description:
-                  "Mock Description: We'd love ot hear from you! Or a very long paragraph here to test the UI display",
-              imageUrl:
-                  "https://dhdbhh0jsld0o.cloudfront.net/m/1ea51560991bcb7d00d0_l",
-            ),
+          PageView.builder(
+            itemBuilder: (context, index) {
+              return Opacity(
+                opacity: 1,
+                child: SurveyCarouselCard(surveyUiModel: cards[index]),
+              );
+            },
+            itemCount: cards.length,
+            controller: PageController(initialPage: 0, viewportFraction: 1),
```

Comment:
we can get rid of default values 🧷 
```suggestion
            controller: PageController(),
```